src/service/subgraph/index.ts

```diff
@@ -14,9 +15,9 @@ export class SubGraphClient {
         }
         
         this.client = new ApolloClient({
-            uri: config.uri,
+            link: new HttpLink({ uri: config.uri, fetch }),
             cache: new InMemoryCache()
-        });
+            });
```

Comment:
Please check the indentation here.

Reply:
* updated the indentation

shall i add prettier dependency, update script in package.json for prettier formatting?
(in a separate PR) ?

@tomarsachin2271

Reply:
Yeah i think it's better to add prettier dependency

Reply:
Merging this PR.. let's add another PR for prettier

Reply:
i will make it on separate PR after approval and merge for this PR